package/yast2-saptune.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
```

Comment:
@angelabriel As it was not yet released, I think it is better to edit the last changelog entry, so we keep only on entry for the entire change. It is easier to read.

Reply:
Ok, this change was requested by Marco in a chat communication. And the new request is done with these changes. But I can revoke this request and change the 'changelog' file again.
Should I  include (and remove) the comment from Peter too, as both are in the update from 1.2 to 1.3?

Reply:
changes done. Hope, it's clearer now.

Reply:
@angelabriel Yes, it is ok, as the last change from Peter wasn't released yet too.